### ISSUE TITLE
"Line contains inconsistent indentation" not correct

### ISSUE THREAD

**Issue by thorge** [janraasch/gulp-coffeelint#58]
Hi there,

I'm experiencing  wrong "Line contains inconsistent indentation" errors.

Simple example to reproduce (method chaining):

```
jQuery.ajax({
  type: 'GET',
  url: 'https://www.google.com',
}).always(function(data) {
  console.log('foo')
});
```

in coffee:

```
jQuery.ajax(
  type: 'GET'
  url: 'https://www.google.com').always (data) ->
  console.log 'foo'
  return
```

results in "Line contains inconsistent indentation"

**Comment by janraasch** [comment]
Hi @thorge

issues such as this should be reported on the [coffeelint](https://github.com/clutchski/coffeelint) repository. `gulp-coffeelint` is only a wrapper making `coffeelint` available as a `gulp`-task.

Thank you :)
